views/index.handlebars

```diff
@@ -1 +1,73 @@
-<h1>Track Finder</h1>
\ No newline at end of file
+
+<nav class="navbar" role="navigation" aria-label="main navigation">
+  <div class="navbar-brand">
+    <a class="navbar-item">
+      <img src="https://webstockreview.net/images/track-clipart.jpg" width="" height=""> <strong>Track Finder</strong>
+    </a>
+
+    <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
+      <span aria-hidden="true"></span>
+      <span aria-hidden="true"></span>
+      <span aria-hidden="true"></span>
+    </a>
+  </div>
+
+  <div id="navbarBasicExample" class="navbar-menu">
+    <div class="navbar-start">
+      <a class="navbar-item">
+        Home
+      </a>
+
+      <a class="navbar-item">
+        Add a Track
+      </a>
+        <a class="navbar-item">
+        Update a Track
+      </a>
+    </div>
+
+    <div class="navbar-end">
+      <div class="navbar-item">
+            <div class="navbar-item has-dropdown is-hoverable">
+        <a class="navbar-link">
+          More
+        </a>
+
+        <div class="navbar-dropdown">
+          <a class="navbar-item" href="https://www.runnersworld.com/">
+            Runners World
+          </a>
+          <a class="navbar-item" href="https://mckirdytrained.com/">
+            McKirdyTrained
+          </a>
+          <a class="navbar-item" href="https://www.mcmillanrunning.com/">
+            McMillan Running Calculator
+          </a>
+                    <a class="navbar-item" href="https://runsmartproject.com/calculator/">
+            Jack Daniels' VDOT
+          </a>
+
+        </div>
+      </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</nav>
+
+<div>
+<h1> Track Finder</h1>
+<h2 class="subtitle">Find a running track near you!</h2>
+<h2 class="subtitle">Go straight, turn left</h2>
+<img id="multiple-runners" src="/assets/images/multiple-runners.png" alt="stick figure runners"/>
+</div>
+<br>
+<div class="field">
+  <label class="label is-large">City</label>
+  <div class="control has-icons-right">
+    <input class="input is-large" type="input" placeholder="Search" id="search">
+    <span class="icon is-small is-right">
+      <i class="fas fa-search"></i>
+    </span>
```

Comment:
can you add a 'submit' or 'search' button here?

Reply:
The search button is in process.  Our wireframe just had the magnifying glass.  Right now it's pushed all the way right.  I was WIP with it being pushed to the left, so it's there-ish.